src/Mod/Arch/ArchWall.py

```diff
@@ -514,19 +672,44 @@ def Activated(self):
         FreeCADGui.doCommand("Arch.joinWalls(FreeCADGui.Selection.getSelection(),delete=True)")
         FreeCAD.ActiveDocument.commitTransaction()
 
+class _Wall(ArchComponent.Component):
+    """The Wall object. 
 
+    Turns a <App::FeaturePython> into a wall object, then uses a
+    <Part::Feature> to create the wall's shape.
 
-class _Wall(ArchComponent.Component):
+    Walls are simple objects, usually vertical, typically obtained by giving a
+    thickness to a base line, then extruding it vertically.
 
-    "The Wall object"
+    Parameters
+    ----------
+    obj: <App::FeaturePython>
+        The object to turn into a wall. Note that this is not the object that
+        forms the basis for the new wall's shape. That is given later.
+    """
 
     def __init__(self, obj):
+        """Initialises the object's properties.
```

Comment:
This docstring can be left empty as its already described in the class docstring.

Reply:
Resolved with e87f173